Unrealm/Tests/UnrealmTests/UnrealmTests.swift

```diff
@@ -0,0 +1,317 @@
+import XCTest
+import Unrealm
+import SnapshotTesting
+import RealmSwift
+import Realm
+
+
+struct Thing: Realmable, Equatable {
+
+    var id: String = "id"
+    var integer: Int = 1
+    var double: Double = 2.5
+    var optionalString: String? = nil
+    var optionalInt: Int? = nil
+    var optionalDouble: Double? = nil
+    var date: Date = .init(timeIntervalSince1970: 0)
+    var optionalDate: Date? = nil
```

Comment:
If you want this to be more comprehensive of primitive types we use in the app, maybe add a Bool and optional Bool to this.